creativecloud/scripts/scripts.js

```diff
@@ -132,7 +132,8 @@ async function loadArticlePromo(getMetadata) {
  * ------------------------------------------------------------
  */
 
-const miloLibs = setLibs(LIBS);
+/* eslint-disable-next-line import/prefer-default-export */
+export const miloLibs = setLibs(LIBS);
 
```

Comment:
I have updated the utils.js to fix the issue here getLibs was returning undefined in this PR https://github.com/orgs/adobecom/discussions/672 As it already has export the I think we can use that to get the libs. Once the above PR merged we can use below code in checkmark-cta and will not need to update scripts.js
<img width="992" alt="Screen Shot 2023-04-27 at 7 04 56 PM" src="https://user-images.githubusercontent.com/69535463/235037397-6d57261f-306d-44e8-977e-f036edac4c9a.png">
.js